src/test/java/Helloworld/HelloWorldTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class HelloWorldTest
+{
+	private static final String s="java";
+	HelloWorld h=new HelloWorld();
```

Comment:
It's a good idea to can re-name variable 'h' to 'classUnderTest' so that we know that class is getting unit tested. You'll understand this when we create objects from other classes that are needed by the 'classUnderTest' directly or indirectly.